.github/workflows/pending-review-notification.yml

```diff
@@ -0,0 +1,35 @@
+name: Send pending review notifications to reviewer on github-discussion
+on:
+  schedule:
+    # Every Tuesday and Thursday at 16:00 UTC.
+    - cron: '0 16 * * 2,4'
+  # GitHub doesn't provide assurance that the scheduled jobs will run on time
+  # (see https://github.community/t/no-assurance-on-scheduled-jobs/133753).
+  # So, we add the workflow_dispatch event here to allow triggering this
+  # workflow manually if needed.
+  workflow_dispatch:
+
+jobs:
+  send_notifications:
+    name: Send pending review notifications
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.8.15'
+          architecture: 'x64'
+      - uses: DubeySandeep/pending-review-notification@v1
```

Comment:
The goal here is to stop depending on `DubeySandeep/pending-review-notification`, so this doesn't look right